sdks/python/apache_beam/coders/standard_coders_test.py

```diff
@@ -30,6 +30,20 @@
 from apache_beam.utils.timestamp import Timestamp
 from apache_beam.transforms.window import IntervalWindow
 
+from nose_parameterized import parameterized
+
+STANDARD_CODERS_YAML = os.path.join(
+    os.path.dirname(__file__), '..', 'tests', 'data', 'standard_coders.yaml')
+
+
+def data(test_yaml):
```

Comment:
Could you add a comment about what this is doing and use a less generic name for the function.